### PR TITLE
Update readme and add issue templates

### PR DESCRIPTION
**Pull request by jdorweiler**
**Reviewer:**

## Description:

- Cleaned up the README and moved all the details about development setup to CONTRIBUTING
- Removed build steps for Safari and added notes that it is no longer supported
- Added CONTRIBUTING that explains how to write a good bug report and feature request
- Added two issue templates for bug reports and feature requests

## Automated tests:
- [ ] Unit tests
- [ ] Integration tests

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,99 +2,27 @@
 
 [![Build Status](https://travis-ci.org/duckduckgo/duckduckgo-privacy-extension.svg?branch=develop)](https://travis-ci.org/duckduckgo/duckduckgo-privacy-extension)
 
-DuckDuckGo Firefox, Chrome, and Safari extensions
+DuckDuckGo Firefox and Chrome extensions
 
-This software is licensed under the terms of the Apache License, Version 2.0 (see LICENSE). Copyright (c) 2012 - 2017 [duckduckgo.com](https://duckduckgo.com)
+This software is licensed under the terms of the Apache License, Version 2.0 (see [LICENSE](LICENSE)). Copyright (c) 2012 - 2018 [duckduckgo.com](https://duckduckgo.com)
```
Comment (kdzwinel) [inline README.md]: 2019 😉BTW Happy New Year 🎇

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
+# Contributing guidelines
+
+# Reporting broken sites
+
+Report broken websites using the "Report Broken Site" link on the extension popup.
+
+# Reporting bugs
+
+- First check to see if the bug has not already been [reported](https://github.com/duckduckgo/duckduckgo-privacy-extension/issues).
+- Make sure that you are using the latest version of the extension.
```
Comment (kdzwinel) [inline CONTRIBUTING.md]: It's probably a good idea to add these notes to the bug report template.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+**How to write a bug report**
```
Comment (kdzwinel) [inline CONTRIBUTING.md]: IMO this section is redundant now that we have a bug reporting template. We can link directly to that template from here like this: 

https://github.com/brainly/html-sketchapp/issues/new?template=html-is-not-imported-correctly-to-sketch.md

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+2. Did this issue start after an upgrade to a new extension version? If so, can you repeat it using a pervious extension version?
+3. Are there any errors in the extension debugging console?
+
+# Feature requests
+
+There are two ways to submit feedback:
+1. You can send anonymous feedback using the "Send feedback" link on the extension's options page.
+2. You can submit your request as a GitHub issue.
+
+**How to write a feature request**
```
Comment (kdzwinel) [inline CONTRIBUTING.md]: ditto, we can just link to feature request template from here

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+2. Get approval from DDG staff before working on it. Since most bug fixes and feature development are handled internally, we want to make sure that your work doesn't conflict with any current projects.
+
+## Testing locally
+
+### Pre-Requisites
+- [Node.js](https://nodejs.org) installation
+- [Grunt](https://www.npmjs.com/package/grunt)
+- Tests use [Selenium Webdriver](http://seleniumhq.github.io/selenium/docs/api/javascript/index.html).
+
+### Set up development environments
```
Comment (kdzwinel) [inline CONTRIBUTING.md]: this section is empty?

**Comment by kdzwinel** [comment]
WDYT about also updating our LICENSE file? Our version of Apache 2.0 license file is not recognized by github, so we don't get those little treats: 

screen shot 2019-01-04 at 11 51 51
screen shot 2019-01-04 at 11 52 00

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+2. `npm install`
+3. `grunt`
+
+**Testing Single Site** `./selenium-test/ratings.js -u https://website.com`
+
+**Testing Top 500** `./selenium-test/ratings.js -n 2` (where n = [1 - 500])
+
+**Testing Multiple Sites** `./selenium-test/ratings.js -f urls.txt` (file should have 1 url on each line)
+
+**Using XVFB** To test on a remote server with XVBF installed, add `-x` flag: `./selenium-test/ratings.js -x -u https://website.com`
```
Comment (andrey-p) [inline CONTRIBUTING.md]: I wonder if it's worth adding a bit about the puppeteer integration tests too?

Comment (jdorweiler) [reply CONTRIBUTING.md]: Added this to the testing section above.

**Pull request merged**